src/wp-admin/options-permalink.php

```diff
@@ -237,101 +237,106 @@
-			<input name="permalink_structure" id="permalink_structure" type="text" value="<?php echo esc_attr( $permalink_structure ); ?>" class="regular-text code" />
-			<div class="available-structure-tags hide-if-no-js">
-				<div id="custom_selection_updated" aria-live="assertive" class="screen-reader-text"></div>
-				<?php
-				$available_tags = array(
-					/* translators: %s: Permalink structure tag. */
-					'year'     => __( '%s (The year of the post, four digits, for example 2004.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'monthnum' => __( '%s (Month of the year, for example 05.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'day'      => __( '%s (Day of the month, for example 28.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'hour'     => __( '%s (Hour of the day, for example 15.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'minute'   => __( '%s (Minute of the hour, for example 43.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'second'   => __( '%s (Second of the minute, for example 33.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'post_id'  => __( '%s (The unique ID of the post, for example 423.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'postname' => __( '%s (The sanitized post title (slug).)' ),
-					/* translators: %s: Permalink structure tag. */
-					'category' => __( '%s (Category slug. Nested sub-categories appear as nested directories in the URL.)' ),
-					/* translators: %s: Permalink structure tag. */
-					'author'   => __( '%s (A sanitized version of the author name.)' ),
-				);
-
-				/**
-				 * Filters the list of available permalink structure tags on the Permalinks settings page.
-				 *
-				 * @since 4.9.0
-				 *
-				 * @param string[] $available_tags An array of key => value pairs of available permalink structure tags.
-				 */
-				$available_tags = apply_filters( 'available_permalink_structure_tags', $available_tags );
-
-				/* translators: %s: Permalink structure tag. */
-				$structure_tag_added = __( '%s added to permalink structure' );
-
-				/* translators: %s: Permalink structure tag. */
-				$structure_tag_already_used = __( '%s (already used in permalink structure)' );
-
-				if ( ! empty( $available_tags ) ) :
-					?>
-					<p><?php _e( 'Available tags:' ); ?></p>
-					<ul role="list">
-						<?php
-						foreach ( $available_tags as $tag => $explanation ) {
-							?>
-							<li>
-								<button type="button"
-										class="button button-secondary"
-										aria-label="<?php echo esc_attr( sprintf( $explanation, $tag ) ); ?>"
-										data-added="<?php echo esc_attr( sprintf( $structure_tag_added, $tag ) ); ?>"
-										data-used="<?php echo esc_attr( sprintf( $structure_tag_already_used, $tag ) ); ?>">
-									<?php echo '%' . $tag . '%'; ?>
-								</button>
-							</li>
+<table class="form-table permalink-structure" role="presentation">
```

Comment:
So something like this: ```<p>
<?php
printf( __( 'Select the permalink structure for your website. A structure that includes the post name makes sharing links to others more understandable.' ) );
?>
</p>
```

Reply:
This might be good at the beginning of the paragraph below the main heading and/or in the Help tab section.

Could you open a separate ticket for editing the text? Ticket 55498 was originally about a missing label, and both of us have already added a few extra changes.

Reply:
https://core.trac.wordpress.org/ticket/55947